SCClassLibrary/Common/Control/Volume.sc

```diff
@@ -64,17 +119,37 @@ Volume {
 	updateSynth {
 		var amp = if(isMuted.not) { volume.dbamp } { 0.0 };
 		var active = amp != 1.0;
+
 		if(active) {
 			if(server.hasBooted) {
-				if(ampSynth.isNil) {
-					ampSynth = Synth.after(server.defaultGroup, defName,
-						[\volumeAmp, amp, \volumeLag, lag, \bus, startBus])
+				if(ampSynth.isPlaying.not) {
+					// Synth.after(server.defaultGroup, defName,
+					// [\volumeAmp, amp, \volumeLag, lag, \bus, startBus])
+
+					server.sendMsg(*group.newMsg(RootNode(server), \addToTail));
+					server.sendMsg(9, //"s_new"
+						defName, ampSynth.nodeID,
+						Node.addActions[\addToTail],
+						group.nodeID,
+						*[\volumeAmp, amp, \volumeLag, lag, \bus, startBus]
+					);
+
+					// also catches stray endings of ampSynth
+					ampSynth.onFree {
+						// "ampSynth was freed".postln;
+						this.volume = 0;
+						ampSynth.isPlaying = false;
+					};
+					ampSynth.isPlaying = true;
```

Comment:
if the ampSynth uses `onFree`, then it is registered anyhow. But you can register it repeatedly without harm.

```


g = Group.basicNew(s, 67);

s.sendMsg(*g.register.newMsg(RootNode(s), \addToTail));

g.isPlaying

// send again
s.sendMsg(*g.register.newMsg(RootNode(s), \addToTail));

g.isPlaying // true

s.freeAll;

g.isPlaying; // false

// send again
s.sendMsg(*g.register.newMsg(RootNode(s), \addToTail));

g.isPlaying // true


// you can register repeatedly:
3.do { g.register };
NodeWatcher.all[\localhost].nodes

```